docs/timezone.rst

```diff
@@ -0,0 +1,143 @@
+Time zones
+==========
+
+Support for time zones is enabled by default. Airflow stores datetime information in UTC internally and in the database.
+ It allows you to run your DAGs with time zone dependent schedules. At the moment Airflow does not them to the end 
```

Comment:
_...Airflow does not them to the end..._

Typo?  Unsure what the word is that's missing though.

Reply:
Good catch, I have to think about that ;-)

Reply:
Hi there I recently upgraded from airflow 1.8 to 1.9 and all of my dags started running in UTC. I tried to set default_timezone = system  and default_timezone = America/Los_Angeles but still the dags were being run in UTC.
I tried to make the dag timezone aware it gave the error - one of the execution /start date was not timezone aware, this is what I tried in my dag.


local_tz = pendulum.timezone('America/Los_Angeles')


default_args = {
    'start_date': datetime(2018, 01, 07, tzinfo=local_tz),
    # 'queue': 'bash_queue',
    # 'end_date': datetime(2016, 1, 1),
}

Reply:
1.9.0 does not support time zones yet. It will arrive in 1.10/2.0 . Before that time it is always assumed airflow is running in UTC as it did not support otherwise.

Reply:
thanks bolkedebruin !